src/main/java/org/sopt/lequuServer/domain/favorite/model/Favorite.java

```diff
@@ -0,0 +1,36 @@
+package org.sopt.lequuServer.domain.favorite.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.sopt.lequuServer.domain.book.model.Book;
+import org.sopt.lequuServer.domain.member.model.Member;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "favorite")
+public class Favorite {
+
+    @Id
+    @Column(name = "favorite_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @ManyToOne
+    @JoinColumn(name = "book_id")
+    private Book book;
+
+    @Builder
+    public Favorite(Member member, Book book) {
+        this.member = member;
+        this.book = book;
+    }
+
+    public static Favorite of(Member member, Book book) {
+        return new Favorite(member, book);
+    }
```

Comment:
```suggestion
    public static Favorite of(Member member, Book book) {
        Favorite favorite = new Favorite(member, book);
        book.addFavorite(favorite);
        member.addFavorite(favorite);
        return favorite;
    }
```
위에서 얘기한 수정 사항입니다~ 이거를 아마 '연관관계 편의성 설정' 뭐 이렇게 불렀던 것 같은데 혹시 더 궁금하면 한번 찾아보세요 :)

Reply:
오호 새롭게 알고갑니당 !!!!!